tests/test_deployments.py

```diff
@@ -131,12 +132,6 @@ async def test_raises_informatively_if_cant_pull(self):
 
 
 class TestDeploymentUpload:
```

Comment:
I'd add some targeted tests for the CLI's `prefect deployment apply --upload ...`.

Reply:
we agreed offline to do this in a follow up PR once I have a plan for mocking remote storage uploads